.github/workflows/continuous-integration-workflow.yml

```diff
@@ -129,6 +129,38 @@ jobs:
               --build-arg nexusAuth=developer:s3cr3t \
               .
 
+  jenkins-agent-nodejs18-centos7:
+    name: Jenkins agent NodeJS 18 (CENTOS7)
+    runs-on: ubuntu-18.04
```

Comment:
would 20.04 make sense?

Reply:
It is `ubuntu-18.04` in all places within this file and I don't know the implications on making the proposed change. Unless the proposed change has a clear benefit or is even a prerequisite for the new build agent, I'd propose to discuss this version change on another issue.

Reply:
My thought was, that this nodejs18 agent might live longer than EOL of ubuntu 18.04.

Reply:
@tbugfinder We could maintain Ubuntu 18.04 for now and take the upgrade as a separate topic/issue